cadquery/occ_impl/exporters/assembly.py

```diff
@@ -99,6 +105,169 @@ def exportAssembly(
     return status == IFSelect_ReturnStatus.IFSelect_RetDone
 
 
+def exportMetaStep(
```

Comment:
Maybe `exportStepMeta` would match the exisitng names better (e.g. `exportCAF`...)

Reply:
Addressed in commit b467f342e6cdf32855054c77c25c45cc4171bdf6